utils/types.ts

```diff
@@ -0,0 +1,30 @@
+import type { VendoJourney, VendoOriginOrDestination } from "@/utils/schemas";
+
+export interface ExtractedData {
+	fromStationId?: string | null;
+	fromStation?: string | null;
+	toStation?: string | null;
+	toStationId?: string | null;
+	date?: unknown;
+	time?: string | null;
+	bahnCard?: string;
+	hasDeutschlandTicket?: boolean;
+	passengerAge?: string;
+	travelClass?: string;
+	class?: number | null;
+	error?: unknown;
```

Comment:
Why is all of this optional? This looks to me like it should at least have some characteristical required properties and be set to optional itself where it's used

e.g.

```typescript
const myFunction(): ExtractedData | null
```